src/Sylius/Behat/Context/Api/Admin/ManagingExchangeRatesContext.php

```diff
@@ -0,0 +1,342 @@
+
+        $this->client->delete('exchange_rates', $exchangeRate->getId());
+    }
+
+    /**
+     * @Then I should see :count exchange rates on the list
+     */
+    public function iShouldSeeExchangeRatesOnTheList(int $count): void
+    {
+        Assert::count($this->client->getCollectionItems(), $count);
+    }
+
+    /**
+     * @Then I should see a single exchange rate in the list
+     * @Then I should( still) see one exchange rate on the list
+     */
+    public function iShouldSeeASingleExchangeRateInTheList(): void
+    {
+        $this->client->index('exchange_rates');
+        Assert::count($this->client->getCollectionItems(), 1);
+    }
+
+    /**
+     * @Then the exchange rate with ratio :ratio between :sourceCurrency and :targetCurrency should appear in the store
+     */
+    public function theExchangeRateWithRatioBetweenAndShouldAppearInTheStore(
+        float $ratio,
+        CurrencyInterface $sourceCurrency,
+        CurrencyInterface $targetCurrency
+    ): void {
+        Assert::true(
+            $this->responseHasExchangeRate($ratio, $sourceCurrency, $targetCurrency),
+            sprintf(
+                'Exchange rate with ratio %s between %s and %s does not exist',
+                $ratio,
+                $sourceCurrency->getName(),
+                $targetCurrency->getName()
+            )
+        );
+    }
+
+    /**
+     * @Then I should see the exchange rate between :sourceCurrency and :targetCurrency in the list
+     * @Then I should (also) see an exchange rate between :sourceCurrency and :targetCurrency on the list
+     */
+    public function iShouldSeeTheExchangeRateBetweenAndInTheList(
+        CurrencyInterface $sourceCurrency,
+        CurrencyInterface $targetCurrency
+    ): void {
+        Assert::notNull(
+            $this->getExchangeRateFromResponse($sourceCurrency, $targetCurrency),
+            sprintf('Exchange rate for %s and %s currencies does not exist', $sourceCurrency, $targetCurrency)
+        );
+    }
+
+    /**
+     * @Then it should have a ratio of :ratio
+     */
+    public function itShouldHaveARatioOf(float $ratio): void
+    {
+        $this->client->index('exchange_rates');
+
+        Assert::true(
+            $this->client->hasItemWithValue('ratio', $ratio),
+            sprintf('ExchangeRate with ratio %s does not exist', $ratio)
+        );
+    }
+
+    /**
+     * @Then /^(this exchange rate) should no longer be on the list$/
+     */
+    public function thisExchangeRateShouldNoLongerBeOnTheList(ExchangeRateInterface $exchangeRate): void
+    {
+        Assert::false(
+            $this->responseHasExchangeRate(
+                $exchangeRate->getRatio(),
+                $exchangeRate->getSourceCurrency(),
+                $exchangeRate->getTargetCurrency()
+            ),
+            sprintf(
+                'Exchange rate with ratio %s between %s and %s still exists',
```

Comment:
`still exists` -> `exists, but it should not.`? We don't know is it "still" or not 😄